cinema/views.py

```diff
@@ -118,12 +131,17 @@ class OrderPagination(PageNumberPagination):
     max_page_size = 100
 
 
-class OrderViewSet(viewsets.ModelViewSet):
```

Comment:
You should make available only 3 actions for MovieSessionViewSet: list, create and retrieve

Reply:
![image](https://github.com/mate-academy/py-token-authentication/assets/114928219/c5c45ee8-8d1d-43e9-bdaa-0d72becbeb11)
nope, i need all actions

> You should make available only 3 actions for MovieSessionViewSet: list, create and retrieve